test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -232,6 +232,15 @@ class BulkRepublisherTest < ActiveSupport::TestCase
       end
     end
 
+    context "for editionable content types, like Contact attached to a WorldwideOrganisation" do
+      test "does not republish all content of the specified type via the Whitehall::Publishing API" do
+        contact = create(:worldwide_organisation, :with_main_office).offices.first.contact
```

Comment:
Then here I wonder if we could stub `can_publish_to_publishing_api?`, just because this currently feels like it requires quite a bit context on what's unique about `WorldwideOrganisation` contacts?

Reply:
That makes the code much more understandable.  Have updated this.